framework/src/controls/ParameterReceiver.C

```diff
@@ -0,0 +1,118 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+// MOOSE includes
+#include "ParameterReceiver.h"
+#include "Function.h"
+
+registerMooseObject("MooseApp", ParameterReceiver);
```

Comment:
```suggestion
registerMooseObject("StochasticToolsApp", ParameterReceiver);
```

Reply:
This in the framework now - I don't believe this is valid ^

Reply:
Oops nevermind